packages/pipeline/src/entities/etherscan_transaction.ts

```diff
@@ -0,0 +1,61 @@
+import { BigNumber } from '@0x/utils';
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+import { bigNumberTransformer, numberToBigIntTransformer } from '../utils';
+
+@Entity({ name: 'etherscan_transactions', schema: 'raw' })
+export class EtherscanTransaction {
+    @PrimaryColumn({ name: 'hash' })
```

Comment:
are we sure this column is a good uniqueness identifier? I'm not sure myself and I know that we've had issues with transaction uniqueness from other sources

Reply:
At the transaction level the hash should be unique. I did a quick query now on the BigQuery Ethereum dataset and this query returned 0 rows:
```select tx.hash, count(*)
from `bigquery-public-data.crypto_ethereum.transactions` as tx
group by 1
having count(*) > 1;```